rmgpy/data/kinetics/family.py

```diff
@@ -1460,7 +1460,8 @@ def applyRecipe(self, reactantStructures, forward=True, unique=True):
             elif isinstance(struct, Group):
                 struct.resetRingMembership()
                 if label in ['1,2_insertion_co', 'r_addition_com', 'co_disproportionation',
-                             'intra_no2_ono_conversion', 'lone_electron_pair_bond']:
+                             'intra_no2_ono_conversion', 'lone_electron_pair_bond',
+                             '1,2_NH3_elimination', '1,3_NH3_elimination']:
```

Comment:
These should be lowercase since we convert the family name to lowercase above.

Reply:
Thanks! done